Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/DXResourceBinding.cpp

```diff
@@ -0,0 +1,59 @@
+#define NOMINMAX
+#define WIN32_LEAN_AND_MEAN
+
+#include "pch.h"
+#include "DXResourceBinding.h"
+#include "DXResourceBinding.g.cpp"
+#include "stdafx.h"
+#include "ORTHelpers.h"
+
+using namespace winrt::Microsoft::AI::MachineLearning;
+
+#undef min
+
+Microsoft::WRL::ComPtr<ID3D12Resource> d3dResource;
+static std::optional<Ort::Session> preprocesingSession;
+static std::optional<Ort::Session> inferenceSession;
+D3D12Quad sample(800, 600, L"D3D12 Quad");
+
+namespace winrt::WinMLSamplesGalleryNative::implementation
+{
+    // Create ORT Sessions and launch D3D window in a separate thread
+	void DXResourceBinding::LaunchWindow() {
```

Comment:
Odd indent.

Reply:
Do you mean to separate the comment and the line with the function definition with a new line?